scripts/apidoc/signature.ts

```diff
@@ -74,9 +74,9 @@ export async function analyzeSignature(
 
   let examples = `${accessor}${methodName}${signatureTypeParametersString}(${signatureParametersString}): ${signature.type?.toString()}\n`;
 
-  const exampleTags = extractRawExamples(signature);
-  if (exampleTags.length > 0) {
-    examples += `${exampleTags.join('\n').trim()}\n`;
+  const exampleTags = extractJoinedRawExamples(signature);
+  if (exampleTags) {
+    examples += exampleTags;
```

Comment:
```suggestion
    examples += `${exampleTags}\n`;
```

Alternatively change the `codeToHtml` method to include that newline

Reply:
is this ok now?